prow/prowjobs/kubeflow/pipelines/kubeflow-pipelines-presubmits.yaml

```diff
@@ -50,13 +50,12 @@ presubmits:
     decorate: true
     spec:
       containers:
-      - image: alpine:3.15.0
+      - image: bash:alpine3.14
         command:
-        - /bin/sh 
+        - bash
```

Comment:
Synced offline, and James explained to me why we cannot have:
```yaml
command:
- ./script.sh
```
because in `bash:alpine3.14` image, the bash location is at `/usr/local/bin/bash` rather than `/bin/bash` which is what the script file has in its shebang.
In this case, I'd suggest either we switch to a different image that has the more common bash location so it saves the trouble and explanation, or we should add a comment here summarize what James found. Otherwise, the next person looking at this command might have the same confusion and go through the same investigation effort.

Reply:
Makes sense, I haven't found an image which is small enough, and is stored at the desired location without explicitly downloading bash. I added command for our findings here. Does it look good to you?

Reply:
Looking good. Thanks!